src/shared.ts

```diff
@@ -54,6 +54,23 @@ const injectScript = (params: null | LoadParams): HTMLScriptElement => {
   return script;
 };
 
+const reloadScript = (params: null | LoadParams): HTMLScriptElement => {
+  const queryString =
+    params && !params.advancedFraudSignals ? '?advancedFraudSignals=false' : '';
+  const script = [...document.getElementsByTagName('script')].filter(
```

Comment:
Isn't it possible that we're altering script element that the merchant added themselves here? Is that what we want to be doing? Should we only affect scripts/DOM elements that we added ourselves?

Further, if this is the approach we want, why not use `findScript` to find the script?

Reply:
In this case, I don't think we are since the filtering is specific to `https://js.stripe.com/v3`. But either way, I'm using @fruchtose-stripe's suggestion of using the existing script and removing it from it's parent node. Didn't think of that!